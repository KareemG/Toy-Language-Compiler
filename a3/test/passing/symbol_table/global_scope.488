
{
    var a : integer
    var b, c : integer

    a := 0
}
