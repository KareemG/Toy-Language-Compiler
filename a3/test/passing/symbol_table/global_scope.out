{
 Symbols:
 b: integer
 a: integer
 c: integer
